README.md

```diff
@@ -1 +1,51 @@
-# Review_Project_Trial
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review_Project_Trial
+
+> One paragraph statement about the project.
+
+![screenshot](./admission.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Html and CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Clone git repo
+
+### Run the index.htm
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@m05e5](https://github.com/m05e5)
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to Arturo Ortega for sharing one of his admission trial work with me
+- Inspiration
+- etc
+
+## 📝 License
```

Comment:
great job :100: you just have to either add this license or remove this part in the readme.md file

Reply:
Well done on the review. The following are some feedback on your comments:

> Hello try to fix all the linters errors 👍

**It is better to use more polite language here and also explain the reason you are making this particular request.**

> you can also give a better pagination to your index.html file to make it look better

**You did well here to use polite language and explain the reason for requesting the change. This will be well understood and appreciated by the student.** 👏 

> great job 💯 you just have to either add this license or remove this part in the readme.md file

**Well done on commending the student but it will be better for you to explain the reason for requesting the change so the students appreciates the review even more.**

Reply:
thanks for the review